weave-js/src/components/PagePanelComponents/Home/Browse3/filters/FilterMetrics.tsx

```diff
@@ -0,0 +1,24 @@
+import {Switch} from '@wandb/weave/components';
```

Comment:
nit: given that this is specifically used in the calls table and not really a general component, i would co-locate it in the page/CallsPage dir